test/lib/write-package.ts

```diff
@@ -219,6 +230,7 @@ describe('writePackage', () => {
           };
         },
         getUser: async (username: string): Promise<false | User> => {
+          username;
```

Comment:
curious why we have this unused username field here?  If it's for the benefit of lint, could we call it `_username`, and shut up. the linter?

Reply:
oh does _username do that?
i put this here to get this comment. or start an async dialog about how to handle the linter here